manifests/install.pp

```diff
@@ -1,6 +1,13 @@
 # Install the needed packages for foreman
 class foreman::install {
 
+  if $foreman::db_manage and $facts['os']['family'] == 'RedHat' {
+    package { 'glibc-langpack-en':
```

Comment:
AFAIK this package does not exist on EL7, so you'd need to guard that somewhere.